src/utils.rs

```diff
@@ -120,12 +120,14 @@ pub trait ToUrl {
 
 impl<P: AsRef<Path> + std::fmt::Debug> ToUrl for P {
     fn to_url(&self) -> Result<Url> {
-        Url::from_file_path(self).or_else(|_| {
-            Err(format_err!(
-                "Failed to convert from path ({:?}) to Url",
-                self
-            ))
-        })
+        Url::from_file_path(self).or_else(|_| (
+            Url::from_str(self.as_ref().to_str().unwrap()).or_else(|_| {
```

Comment:
`.unwrap()` on a `None` will be a hard crash.

Please use `Url::from_str(&self.as_ref().to_string_lossy())` instead.